ndb/src/google/cloud/ndb/_future.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A Future class."""
+
+_NOT_COMPUTED = object()
```

Comment:
We've run into issues before where an `object()` doesn't work across process boundaries (e.g. in Pub / Sub). Would it be possible to instead just use a `uuid4()`?

Reply:
Not intended to cross a process boundary. IPC is being done by gRPC--this would never be serialized to go out over a wire.